.travis.yml

```diff
@@ -57,9 +53,21 @@ script: bash build_tools/travis/test_script.sh
 after_success: source build_tools/travis/after_success.sh
 
 notifications:
-    slack:
-      rooms:
-        - tgsmith61591-gh:mWyfLXzxRAHCKD2QRHWvHZaX#pyramid-cicd
-      on_success: always
-      on_failure: always
-      on_pull_requests: true
+  slack:
+    rooms:
+    - tgsmith61591-gh:mWyfLXzxRAHCKD2QRHWvHZaX#pyramid-cicd
+    on_success: always
+    on_failure: always
+    on_pull_requests: true
+
+deploy:
+  #server: https://test.pypi.org/legacy/ # Upload to pypi test server
+  distributions: "bdist_wheel" # Upload tar.gz and .whl files.
+  skip_upload_docs: true
+  skip_cleanup: true
+  on:
+    tags: true # Upload on tagged releases
```

Comment:
What about on branch `master`? Or is the `tags` technically proxy for that, since we'd never tag a non `master` release?